truss/tests/test_model_inference.py

```diff
@@ -33,6 +44,49 @@ def test_pytorch_init_arg_validation(
         validate_provided_parameters_with_model(pytorch_model_with_init_args, {})
 
 
+def test_slow_load_model():
+    with open(
+        f"../test_data/models/slow_load/{CONFIG_FILE}", encoding="utf-8"
+    ) as config_file:
+        config = yaml.safe_load(config_file)
+        model = ModelWrapper(config)
```

Comment:
can we instantiate an inference server here instead of unwraping the code in it for the test?

Reply:
`inference_server.py` isn't really built to be called from within another python class, but there's no reason why i can't structure it to work that way. A good idea to not duplicate code there and I'll see if I can rework.